code/game/objects/items/robot/items/generic.dm

```diff
@@ -15,10 +15,17 @@
 /obj/item/borg/stun
 	name = "electrically-charged arm"
 	icon_state = "elecarm"
+	var/stamina_damage = 60 //Same as normal batong
 	/// Cost to use the stun arm
-	var/charge_cost = 1000
+	var/charge_cost = 200
+	var/cooldown_check = 0
+	/// cooldown between attacks
+	var/cooldown = 4 SECONDS // same as baton
```

Comment:
If this is intended to remain contemporary with a baton we should probably have a define somewhere which is shared by both items, so that changing one changes both.

Reply:
It's intended to be equal to baton for now. It seems its a good point for now, since I have seen no major problems during TM, but it will likely need some tweaks, so I'm not going to tie it to batong for now.